src/components/DpDataTable/DpDataTable.vue

```diff
@@ -687,7 +687,7 @@ export default {
       // Remove styles set by initialMaxWidth and initialWidth after copying rendered width into th styles
       if (this.isResizable) {
         Array.from(tableHeaderElements).forEach(th => {
-          if (th.firstChild) {
+          if (th.firstChild && th.firstChild.nodeType === 1) {
```

Comment:
Just out of curiosity: what else can nodeType be?

Reply:
Here is a [list of all node types](https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType). Only node type `element` (1) can have the style attribute.

Reply:
But in This Case I mean. When is it no Element?

Reply:
I see, it can be a text node